test/Riok.Mapperly.Tests/Mapping/EnumTest.cs

```diff
@@ -6,6 +6,33 @@ namespace Riok.Mapperly.Tests.Mapping;
 [UsesVerify]
 public class EnumTest
 {
+    [Fact]
+    public void EnumToOtherEnumWithExplicitEnumMapping()

```

Comment:
Add tests for the following cases:
* The same source specified multiple times (should diagnostic)
* The same target specified multiple times